### PR TITLE
Fix getVar() method when trying to get JSON variable that does not exists

### LINKED ISSUE #4906: Bug: getVar does not return null when item doesn't exist
**Describe the bug**
getVar() doesn't return null when the item I'm attempting to retrieve does not exist

**CodeIgniter 4 version**
4.1.3

**Affected module(s)**
IncomingRequest?

**Expected behavior, and steps to reproduce if appropriate**
Inside of any controller, dump the contents of getVar() with any parameter you'd like:
```php
public function update() {
	var_dump($this->request->getVar('description'));
}
```

Now, using Postman, I made a raw JSON request (HTTP PUT) with the following body (empty works, too):
```json
{
	"name": "test"
}
```

Expected behaviour: _NULL_ should be printed, as per the [documentation](https://codeigniter4.github.io/userguide/incoming/incomingrequest.html?highlight=getvar#getVar)
Actual behaviour: _string(0) ""_ is printed.

**Context**
 - OS: Windows 10
 - Web server: Apache
 - PHP version: 7.4.3

### PR DESCRIPTION
**Pull request by michalsn**
**Description**
Fixes #4906

**Checklist:**
- [x] Securely signed commits
- [x] Unit testing, with >80% coverage
- [x] Conforms to style guide

### CODE REVIEW

**Review by paulbalandan — APPROVED** [review]

File: tests/system/HTTP/IncomingRequestTest.php [hunk 1]
```diff
@@ -318,6 +318,7 @@ public function testCanGetAVariableFromJson()
         $request = new IncomingRequest($config, new URI(), $json, new UserAgent());
 
         $this->assertEquals('bar', $request->getJsonVar('foo'));
+        $this->assertSame(null, $request->getJsonVar('notExists'));
```
Comment (MGatner) [inline tests/system/HTTP/IncomingRequestTest.php:321]:
```suggestion
        $this->assertNull($request->getJsonVar('notExists'));
```

File: tests/system/HTTP/IncomingRequestTest.php [hunk 2]
```diff
@@ -373,6 +374,7 @@ public function testGetVarWorksWithJson()
 
         $this->assertEquals('bar', $request->getVar('foo'));
         $this->assertEquals('buzz', $request->getVar('fizz'));
+        $this->assertSame(null, $request->getVar('notExists'));
```
Comment (MGatner) [inline tests/system/HTTP/IncomingRequestTest.php:377]:
```suggestion
        $this->assertNull($request->getVar('notExists'));
```

**Review by MGatner — CHANGES_REQUESTED** [review]
Nice fix. Quick hardening of assertions.

**Comment by michalsn** [comment]
The failing test is irrelevant to the PR.

**Review by MGatner — APPROVED** [review]

**Pull request merged**